lib/vm/src/lib.rs

```diff
@@ -58,24 +58,26 @@ pub use crate::vmoffsets::{TargetSharedSignatureIndex, VMOffsets};
 /// Version number of this crate.
 pub const VERSION: &str = env!("CARGO_PKG_VERSION");
 
-/// A safe wrapper around `VMFunctionBody`
+/// A safe wrapper around `VMFunctionBody`.
 #[derive(Clone, Copy, Debug)]
 #[repr(transparent)]
-pub struct FunctionBodyPtr(pub *const [VMFunctionBody]);
+pub struct FunctionBodyPtr(pub *const VMFunctionBody);
 
 impl std::ops::Deref for FunctionBodyPtr {
-    type Target = *const [VMFunctionBody];
+    type Target = *const VMFunctionBody;
 
     fn deref(&self) -> &Self::Target {
         &self.0
     }
 }
 
 /// # Safety
-/// TODO:
+/// The VMFunctionBody that this points to is opaque, so there's no data to
+/// read or write through this pointer. This is essentially a usize.
 unsafe impl Send for FunctionBodyPtr {}
 /// # Safety
-/// TODO:
+/// The VMFunctionBody that this points to is opaque, so there's no data to
+/// read or write through this pointer. This is essentially a usize.
```

Comment:
It's surprising that `Send` isn't automatically implemented for types that are both `Sync` and `Copy`